Source/WebCore/page/LocalFrameView.h

```diff
@@ -1017,6 +1022,7 @@ class LocalFrameView final : public FrameView {
     ScrollPinningBehavior m_scrollPinningBehavior { ScrollPinningBehavior::DoNotPin };
     SelectionRevealMode m_selectionRevealModeForFocusedElement { SelectionRevealMode::DoNotReveal };
     std::unique_ptr<ScrollAnchoringController> m_scrollAnchoringController;
+    WeakHashSet<ScrollAnchoringController> m_anchorsNeedingUpdate;
```

Comment:
This is a set of controllers, not a set of anchors. What does it represent - controllers for subframes, or overflow scrollers?

Reply:
Updated the name to m_scrollAnchoringControllersNeedingUpdate. This represents the set of scroll anchoring controllers needing position updates of the subscrollers for the FrameView (as well as the controller for the frame as well).

Reply:
I think it would be better to just store references to ScrollableAreas (we have a set already). And we need to think about how this works with site isolation.

Reply:
Updated to ScrollableAreaSet (although we need to keep this separate from m_scrollableAreas since not every scrollable area is going to need a scroll position update for each frame).